packages/react-router/src/Link.tsx

```diff
@@ -1,17 +1,30 @@
 import React, { useContext, useEffect } from "react";
 import { Link as RouterLink, LinkProps as RouterLinkProps } from "react-router-dom";
 import { RouterContext } from "./context/RouterContext";
+import PropTypes from "prop-types";
```

Comment:
I think we're safe to remove this, the props are checked with TS.

Reply:
Yea, my bad.  Had it removed, then forgot to clean it up after I undid something else I was working on.